lib/compress.js

```diff
@@ -2977,4 +2977,26 @@ merge(Compressor.prototype, {
         return self;
     });
 
+    OPT(AST_TemplateString, function(self, compressor){
+        var segments = [];
+        for (var i = 0; i < self.segments.length; i++) {
+            if (self.segments[i] instanceof AST_Node) {
+                var result = self.segments[i].evaluate(compressor);
+
+                if (segment.length === 1) {
+                    segments.push(result[0]);
+                    continue;
+                }
+
+                // There should always be a previous and next segment if segment is a node
+                segments[segments.length - 1] = segments[segments.length - 1] + result[1] + self.segments[++i];
```

Comment:
`segments` is more fitting. `output` will cause confusion with the output object in `lib/output.js`.